src/java.xml.crypto/share/legal/santuario.md

```diff
@@ -223,5 +223,5 @@ The development of this software was partly funded by the European
 Commission in the <WebSig> project in the ISIS Programme.
 
 This product contains software that is
-copyright (c) 2021, Oracle and/or its affiliates.
+copyright (c) 2021, 2023, Oracle and/or its affiliates.
```

Comment:
Why not 2024?

Reply:
This is copy of the [LICENSE](https://github.com/apache/santuario-xml-security-java/blob/main/NOTICE) file from Santuario.